R/database.R

```diff
@@ -61,38 +63,48 @@ get_ssm_by_sample = function(this_sample_id,
 
   this_unix_group = pull(these_samples_metadata,unix_group)
   this_genome_build = pull(these_samples_metadata,genome_build)
+  this_seq_type = pull(these_samples_metadata,seq_type)
+  this_pair_status = pull(these_samples_metadata,pairing_status)
   if(verbose){
     print(paste("group:",this_unix_group,"genome:",this_genome_build))
   }
+  # get unmatched normal if necessary
+  if(pair_status == "unmatched"){
```

Comment:
Another comment I had, would it make more sense to have `pair_status == "unmatched"` set to `this_pair_status == "unmatched"` (to match the pull argument above) or even `pairing_status == "unmatched"` (to match the variable from get_gamble_metadata)? when executing function line by line it complains about the object "pair_status" is not found. 

After updating this line to `if(this_pairing_status == "unmatched")` and flavour = "clustered" the function runs the intended way.

Reply:
I had avoided using parameter names that were identical to columns because that caused namespace problems in the function. e.g. if we want to refer to the column pair_status once we have a variable named pair_status how do we deal with that? Maybe there's an obvious way I'm just oblivious to.